src/lib/assets/Assets.stories.svelte

```diff
@@ -60,7 +60,8 @@
       <div class="flex flex-col items-start justify-start p-4">
         <h2 class="mb-2 text-lg font-semibold">{name}</h2>
         {#if Icon === OpenClose}
-          <aside>* Click me!</aside>
+          <input type="checkbox" class="peer hidden" id="mobile-menu-toggle" />
```

Comment:
The open close now all triggers off of a checkbox. Why?

1. CSS rendering is always faster than js
2. Works without JS